models/reports.yaml

```diff
@@ -8,85 +8,103 @@ models:
       html: |
         <!DOCTYPE html>
         <html lang="en">
-
         <head>
-            <title>
-                ID Stitcher Report
-            </title>
+            <title>ID Stitcher Report</title>
             <style>
-                .table-container {
-                    margin-bottom: 3em;
-                }
+            .table-container {
+                margin-bottom: 3em;
+            }
 
-                .table-container table {
-                    border-collapse: collapse;
-                    width: 50%;
-                }
+            .table-container table {
+                border-collapse: collapse;
+                width: 50%;
+            }
 
-                .table-container th,
-                .table-container td {
-                    border: 1px solid #dddddd;
-                    text-align: left;
-                    padding: 0.5em;
-                }
+            .table-container th,
+            .table-container td {
+                border: 1px solid #dddddd;
+                text-align: left;
+                padding: 0.5em;
+            }
 
-                .table-container th {
-                    background-color: #f2f2f2;
-                }
+            .table-container th {
+                background-color: #f2f2f2;
+            }
 
-                img {
-                    max-width: 100%;
-                    height: auto;
-                    display: block;
-                    margin: 1em auto;
-                }
+            img {
+                max-width: 100%;
+                height: auto;
+                display: block;
+                margin: 1em auto;
+            }
             </style>
         </head>
 
-        <body style="text-align:center">
-            <h1 style="color: green">
-                Testing
-            </h1>
+        <body style="text-align: center">
+            <h1 style="color: green">Testing</h1>
+            {% set query_models = ["models/clusters", "models/overall_stats", "models/clusters_sorted_by_size", "models/clusters_sorted_by_dist", "models/top_max_degree_nodes", "models/edges_graph", "models/edges_biggest_cluster", "models/id_type_count", "models/id_count"] %}
+            {% for qmodel in query_models %}{% with _ = this.DeRef(qmodel) %}{% endwith %}{% endfor %}
```

Comment:
This line is not required because you already de-refing below.

Reply:
addressed